src/components/draggable-control.js

```diff
@@ -20,17 +20,26 @@
 import PropTypes from 'prop-types';
 import BaseControl from './base-control';
 
+import type {MjolnirEvent} from 'mjolnir.js';
+
 const propTypes = Object.assign({}, BaseControl.propTypes, {
   draggable: PropTypes.bool,
   onDrag: PropTypes.func,
   onDragEnd: PropTypes.func,
-  onDragStart: PropTypes.func
+  onDragStart: PropTypes.func,
+  // Offset from the left
+  offsetLeft: PropTypes.number,
+  // Offset from the top
+  offsetTop: PropTypes.number
```

Comment:
Moving the `offset*` props back into draggable control felt better than overriding _getDragLngLat in Marker. If there are other classes extending DraggableControl in the future, they may also want to support offsets.